azure-mgmt-consumption/azure/mgmt/consumption/__init__.py

```diff
@@ -1,9 +1,5 @@
 # coding=utf-8
 # --------------------------------------------------------------------------
-# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This is not generated using the official command line, otherwise this should not be in the diff. Please share how you generated this client,

Reply:
@lmazuel I have reverted changes in mgmt_testcase.py

Reply:
@lmazuel command used to generate SDK 
autorest --version=latest --python --azure-arm=true --payload-flattening-threshold=2 --input-file=C:\AIP\SDK\AzureRestAPISpecs\specification\consumption\resource-manager\Microsoft.Consumption\stable\2018-01-31\consumption.json

Reply:
@kjeur Please use:
autorest specification/consumption/resource-manager/readme.md --python --python-sdks-folder=C:\azure-sdk-for-python --python-mode=update

See:
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/consumption/resource-manager/readme.md#python